### PR TITLE
WT-7708 Add an assert to ensure stable timestamp is not moved during prepared commit and fix test format

### PR DESCRIPTION
**Pull request by quchenhao**

### CODE REVIEW

**Comment by quchenhao** [comment]
Stress test is passing with the change.

**Review by keithbostic — APPROVED** [review]
lgtm

File: test/format/ops.c [hunk 1]
```diff
@@ -505,6 +507,10 @@ commit_transaction(TINFO *tinfo, bool prepared)
     session = tinfo->session;
 
     ts = 0; /* -Wconditional-uninitialized */
+
+    if (prepared)
+        lock_readlock(session, &g.prepare_commit_lock);
+
```
[the remark this answers is not in the document]
Comment (quchenhao) [reply test/format/ops.c:513]: I think for prepared case, timestamp must be configured. Also we need to ensure when we are committing, the stable timestamp is not moved so the unlocking has to be after commit.

[the remark this answers is not in the document]
Comment (keithbostic) [reply test/format/ops.c:513]: Yes; is it easiest to duplicate the actual line of code that does the commit?

[the remark this answers is not in the document]
Comment (quchenhao) [reply test/format/ops.c]: Yes, that's true. Fixed it.

**Pull request merged**